test_e2e.sh

```diff
@@ -3250,6 +3250,21 @@ function e2e::github_https() {
     assert_file_exists "$ROOT/link/LICENSE"
 }
 
+##############################################
+# Test github app auth
+##############################################
+function e2e::github_app_auth() {
```

Comment:
Put this with the other `e2e::auth_*` tests and name it `e2e::auth_github_app`.

Reply:
Done